coulomb-physical-constants/src/main/scala/coulomb/physicalconstants/package.scala

```diff
@@ -0,0 +1,495 @@
+      DerivedUnit[BohrMagneton, Joule %/ Tesla](coef = num / denom, name = "bohr-magneton", abbv = "μB")
+    }
+
+    trait NuclearMagneton
+    implicit val defineNuclearMagneton = {
+      // original value = 5.0507837461(15)×10−27 J⋅T−1
+      val denom = Rational(10).pow(27)
+      val num = Rational(50507837461L) / Rational(10).pow(10)
+      DerivedUnit[NuclearMagneton, Joule %/ Tesla](coef = num / denom, name = "nuclear-magneton", abbv = "μₙ")
+    }
+
+    trait FineStructureConstant
+    implicit val defineFineStructureConstant = {
+      // original value = 7.2973525693(11)×10−3
+      val denom = Rational(10).pow(3)
+      val num = Rational(72973525693L) / Rational(10).pow(10)
+      DerivedUnit[FineStructureConstant, Unitless](coef = num / denom, name = "fine-structure-constant", abbv = "α")
+    }
+
+    trait InverseFineStructureConstant
+    implicit val defineInverseFineStructureConstant = {
+      // original value = 137.035999084
+      val coef = Rational(137035999084L) / Rational(10).pow(9)
+      DerivedUnit[InverseFineStructureConstant, Unitless](coef = coef, name = "inverse-fine-structure-constant", abbv = "α⁻ⁱ")
+    }
+
+    trait ElectronMass
+    implicit val defineElectronMass = {
+      // original value = 9.1093837015(28)×10−31 kg
+      val denom = Rational(10).pow(31)
+      val num = Rational(91093837015L) / Rational(10).pow(10)
+      DerivedUnit[ElectronMass, Kilogram](coef = num / denom, name = "electron-mass", abbv = "mₑ")
+    }
+
+    trait ProtonMass
+    implicit val defineProtonMass = {
+      // original value = 1.67262192369(51)×10−27 kg
+      val denom = Rational(10).pow(27)
+      val num = Rational(167262192369L) / Rational(10).pow(11)
+      DerivedUnit[ProtonMass, Kilogram](coef = num / denom, name = "proton-mass", abbv = "mₚ")
+    }
+
+    trait NeutronMass
+    implicit val defineNeutronMass = {
+      // original value = 1.67492749804(95)×10−27 k
+      val denom = Rational(10).pow(27)
+      val num = Rational(167492749804L) / Rational(10).pow(11)
+      DerivedUnit[NeutronMass, Kilogram](coef = num / denom, name = "neutron-mass", abbv = "mₚ")
+    }
+
+    trait BohrRadius
+    implicit val defineBohrRadius = {
+      // original value = 5.29177210903(80)×10−11 m
+      val denom = Rational(10).pow(11)
+      val num = Rational(529177210903L) / Rational(10).pow(11)
+      DerivedUnit[BohrRadius, Meter](coef = num / denom, name = "bohr-radius", abbv = "a₀")
+    }
+
+    trait ClassicalElectronRadius
+    implicit val defineClassicalElectronRadius = {
+      // original value = 2.8179403262(13)×10−15 m
+      val denom = Rational(10).pow(15)
+      val num = Rational(28179403262L) / Rational(10).pow(10)
+      DerivedUnit[ClassicalElectronRadius, Meter](coef = num / denom, name = "classical-electron-radius", abbv = "rₑ")
+    }
+
+    trait ElectronGFactor
+    implicit val defineElectronGFactor = {
+      // original value = −2.00231930436256(35)
```

Comment:
This is a problem because you cannot pass a negative as coefficient.
I don't know if there is a way around it or whether I should just remove this constant

Reply:
We should be able to negate it up above in definition of `electronGFactor`